src/replica/replica_stub.cpp

```diff
@@ -2239,33 +2204,11 @@ void replica_stub::register_ctrl_command()
 #elif defined(DSN_USE_JEMALLOC)
         register_jemalloc_ctrl_command();
 #endif
-        // TODO(yingchun): use http
-        _cmds.emplace_back(::dsn::command_manager::instance().register_command(
-            {"replica.max-concurrent-bulk-load-downloading-count"},
-            "replica.max-concurrent-bulk-load-downloading-count [num | DEFAULT]",
-            "control stub max_concurrent_bulk_load_downloading_count",
-            [this](const std::vector<std::string> &args) {
-                std::string result("OK");
-                if (args.empty()) {
-                    result = "max_concurrent_bulk_load_downloading_count=" +
-                             std::to_string(_max_concurrent_bulk_load_downloading_count);
-                    return result;
-                }
-
-                if (args[0] == "DEFAULT") {
-                    _max_concurrent_bulk_load_downloading_count =
-                        _options.max_concurrent_bulk_load_downloading_count;
-                    return result;
-                }
-
-                int32_t count = 0;
-                if (!dsn::buf2int32(args[0], count) || count <= 0) {
-                    result = std::string("ERR: invalid arguments");
-                } else {
-                    _max_concurrent_bulk_load_downloading_count = count;
-                }
-                return result;
-            }));
+        _cmds.emplace_back(::dsn::command_manager::instance().register_int_command(
```

Comment:
Should we add validator to ensure that `max_concurrent_bulk_load_downloading_count` > 0 ?

Reply:
register_int_command() has a default validator to ensure it >= 0. I updated the code to check the FLAGS_*